express/blocks/columns/columns.css

```diff
@@ -120,4 +140,40 @@ main .columns .social.icon {
   main .columns.fullsize + h2 {
     margin-top: 0;
   }
+}
+
+@media (max-width: 899px) {
+  main .columns .hero-animation-overlay {
+    order: -1;
+  }
+
+  main .new-template-list-sixcols-container > div > .button-container {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 40px;
```

Comment:
The floating button container should also have a vertical linear gradient background (transparent to white).